server/internal/client/ollama/registry.go

```diff
@@ -97,23 +112,43 @@ func (e *Error) Error() string {
 	return fmt.Sprintf("registry responded with status %d: %s %s", e.Status, e.Code, e.Message)
 }
 
+func (e *Error) LogValue() slog.Value {
+	return slog.GroupValue(
+		slog.Int("status", e.Status),
+		slog.String("code", e.Code),
+		slog.String("message", e.Message),
+	)
+}
+
 // UnmarshalJSON implements json.Unmarshaler.
 func (e *Error) UnmarshalJSON(b []byte) error {
 	type E Error
-	var v struct{ Errors []E }
+	var v struct {
+		// Single error
+		Code  string
+		Error string
+
+		// Multiple errors
+		Errors []E
+	}
 	if err := json.Unmarshal(b, &v); err != nil {
 		return err
 	}
+	if v.Error != "" {
+		// Single error case
+		e.Code = v.Code
+		e.Message = v.Error
+		return nil
+	}
 	if len(v.Errors) == 0 {
 		return fmt.Errorf("no messages in error response: %s", string(b))
 	}
 	*e = Error(v.Errors[0]) // our registry only returns one error.
 	return nil
 }
 
-// TODO(bmizerany): make configurable on [Registry]
 var defaultName = func() names.Name {
-	n := names.Parse("ollama.com/library/_:latest")
+	n := names.Parse("registry.ollama.ai/library/_:latest")
```

Comment:
This default is matching what we have somewhere else in the codebase? Just to confirm.

Reply:
Yes. Unfortunatly we have not moved from this to ollama.com, and so if we change it now without _fallback_ mechanisms, users will end up with `ollama la` showing things like:

```
registry.ollama.ai/library/llama3.2
llama3.2
```

where the latter is using the new `ollama.com` and the former is considered "unofficial" by our client. We can address this later. It's not a huge blocker. Mostly vanity.